packages/mariadb/test/test.coffee

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
I don't understand why you are copying a "test.sample.coffee", the environment is always the same, it doesn't not depends and the host machine and his always executed entirely. This is not like in Nikita where only a subset of the tests may be executed.